ansible/roles/docker/templates/service.j2

```diff
@@ -12,10 +12,13 @@ Type=notify
 TimeoutStartSec=0
 Delegate=yes
 KillMode=process
-LimitNOFILE=infinity
+LimitNOFILE=1048576
```

Comment:
Do we know the kernel setting?

Reply:
I have not checked the kernel setting, this should be the default setting from the docker package